src/Packetery/Module/Options.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Packetery\Module;
+
+class Options {
```

Comment:
Udělal bych to jako Packetery\Module\Options\OptionNames a odstranil bych z názvů konstant zbytečné prefixy.